index.Rmd

```diff
@@ -60,7 +60,7 @@ SWWdf<-occ2df(SWW)
 Spocc downloads data via "occ" and converts to a dataframe via "occ2df".
 
 # Results
-My figures are not currently displayed, a placeholder image is below. The final project will have a slider built in Shiny that allows the user to control the timeframe of the data displayed.
+My figures are not currently displayed, a placeholder image is below. The final project will have a slider built in Shiny that allows the user to control the timeframe of the data displayed. 
```

Comment:
The placeholder map shows what looks like distributions of various invasive species as well as the date they became established.  Will temperature change over time be a layer or option for display on the final?  I ask because your central question/hypothesis mentions a correlation with mean temperature rise.